catalog/sync_test.go

```diff
@@ -182,6 +185,16 @@ func createInstanceInAWS(a *awssd.Client, serviceID string) error {
 			"FUBAR":             "BARFU",
 		},
 	})
+	//if err != nil {
+	//	return err
+	//}
+	//
+	//time.Sleep(30 * time.Second) // This is a hack to wait for the instance to be created
+	//_, err = a.UpdateInstanceCustomHealthStatus(context.TODO(), &awssd.UpdateInstanceCustomHealthStatusInput{
+	//	InstanceId: &serviceID,
+	//	ServiceId:  &serviceID,
+	//	Status:     awssdtypes.CustomHealthStatusHealthy,
+	//})
```

Comment:
This is probably a useful bit of the test, but I didn't want to spend the time polishing since the health status syncing is unimplemented from Consul --> AWS.

Reply:
Are you meaning to leave this in but commented out for the future if we add health status syncing?

Reply:
Yes. As someone new to this AWS service it wasn't intuitive to figure out how to update the health status, so I think it will be a useful bit if/when health syncing is enabled.